README.md

```diff
@@ -175,6 +175,7 @@ For manual upgrade with `git` (requires git v1.7.10+):
 ```sh
 (
   cd "$NVM_DIR"
+  git pull
```

Comment:
This command shouldn’t ever be used, since it merges by default - the fetch should be sufficient, after #1793.